docs/how-to-guides/upgrade-openebs.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Upgrading OpenEBS control plane and data plane components
+weight: 10
+---
+
+## Introduction
+
+This guide provides step-by-step instructions on how to upgrade
+[OpenEBS](https://openebs.io/) control plane and data plane components from one
+version to another.
+
+To get the current version installed, execute:
+```bash
+kubectl get pods -n openebs -o jsonpath='{.items[*].metadata.labels.openebs\.io/version}'
+```
+
+Refer to the release notes of the Lokomotive release to get the version to
```

Comment:
Maybe we can put a link here to the release notes?

Reply:
We don't have a release notes document if I understand correctly.

Reply:
We have a changelog, I think we should point there.

Reply:
ah yes, of course :facepalm: